scripts/pretty-printers/gdb/pretty_printers.py

```diff
@@ -1,4 +1,21 @@
 import gdb
+import json
```

Comment:
:no_entry_sign: please don't put prettty printing changes into the same commit as the body of the work

Reply:
Or even in this PR??

Reply:
I've not reviewed these changes on the assumption they are in error!

Reply:
Oops